astroid/brain/brain_attrs.py

```diff
@@ -10,7 +10,9 @@
 from astroid.nodes.node_classes import AnnAssign, Assign, AssignName, Call, Unknown
 from astroid.nodes.scoped_nodes import ClassDef
 
-ATTRIB_NAMES = frozenset(("attr.ib", "attrib", "attr.attrib", "attr.field", "field"))
+ATTRIB_NAMES = frozenset(
+    ("attr.ib", "attrib", "attr.attrib", "attr.field", "attrs.field", "field")
```

Comment:
I think we could discuss if `attrib` and `field` should be in this list but that ship has sailed I think.

Thanks for removing `define`! 👍

Reply:
I think you're right, it's also a problem.
```suggestion
    ("attr.ib", "attr.attrib", "attr.field", "attrs.field")
```
Suggestion done  in order to check if it's causing test fails. Imo we can most probably not infer the value as aliasing of attrs must be rare considering it's a small name, but we also *should* infer.

Reply:
looks like it breaks the existing `test_attr_transform` as well as the newly added `test_attrs_transform`.
```py
        @attrs
        class Bah:
            d = attrib(attr.Factory(dict))
```

Also there's a regression test in `pylint` that also relies on the existing `attrib`; https://github.com/PyCQA/pylint/blob/main/tests/functional/r/regression/regression_4439.py

Reply:
@Pierre-Sassoulas Let's revert then. Thanks @jacobbogdanov for checking this out.